grammar.js

```diff
@@ -175,7 +175,7 @@ module.exports = grammar({
 				/[^xu\d]/,
 				/\d{1,3}/,
 				/x[\da-fA-F]{2}/,
-				/u{[\da-fA-F]+}/,
+				/u\{[\da-fA-F]+\}/,
```

Comment:
This is the only functional (and necessary) change; the rest are generated bindings.